DDAlign/src/DDAlignForwardCall.cpp

```diff
@@ -0,0 +1,32 @@
+//==========================================================================
+//  AIDA Detector description implementation for LCD
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : M.Frank
+//
+//==========================================================================
+
+// Framework includes
+#include "DDAlign/DDAlignForwardCall.h"
+#include "DD4hep/ConditionsData.h"
+#include "DD4hep/Printout.h"
+
+using namespace DD4hep;
+using namespace DD4hep::Conditions;
+
+/// Interface to client Callback in order to update the condition
+Condition
+Alignments::DDAlignForwardCall::operator()(const ConditionKey& key, const UpdateContext& context)
+{
+  const Data::Delta delta;
```

Comment:
error: default initialization of an object of const type 'const Data::Delta' (aka 'const DD4hep::Alignments::Delta') without a user-provided default constructor